sections/image-banner.liquid

```diff
@@ -33,11 +33,13 @@
   if section.settings.image_behavior == 'ambient'
     assign half_width = '60vw'
     assign full_width = '120vw'
+    assign max_width = 4608
     assign stacked_sizes = '(min-width: 750px) 60vw, 120vw'
-    assign widths = '450, 660, 900, 1320, 1800, 2136, 2400, 3600, 7680'
+    assign widths = '450, 660, 900, 1320, 1800, 2136, 2400, 3600, 4608'
```

Comment:
Let me know here if there was a specific reason for this 🤔 Because the section is full width I could see how very wide screen could be an issue but the gap between `3600` and `7680` seemed odd. 
cc: @kjellr

Reply:
fwiw even if 7680 (or anything remotely near that) was called for by the viewport, I'm not sure we would want to allow the requesting of such a large asset.

Reply:
Yeah, I think we're probably okay losing that value. 👍